package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "author": "Andrew Balmos <andrew@balmos.org> (https://github.com/abalmos)",
   "license": "Apache 2.0",
+  "scripts": {
+    "test": "./istantestd.sh"
```

Comment:
I like it with mocha because I can run mocha in watch mode. We don't have that so I guess not.